dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/MemoryUsageTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+import com.google.common.collect.ImmutableList;
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MemoryUsageTest {
```

Comment:
Do you still need this test?

Reply:
I'm not sure. What do you think? I think we need something like this, though perhaps I should cut pieces of it, or move it to a manual test. Then again, it would be nice to notice if we do something that increases the size past what Java can handle.

Reply:
I think we need just one test in ClassPathBuilder to ensure that ClassPathBuildet can resolve the class path of the hcatalog artifact.